VegetableSalad/VegetableSalad/Services/CreatePlant.cs

```diff
@@ -0,0 +1,47 @@
+﻿
+using VegetableSalad.Services.Abstraction;
+
+namespace VegetableSalad.Services
+{
+    internal class CreatePlant : ICreatePlant
+    {
+        public string PlantType { get; } = "Vegetable";
+        public string Name { get; set; }
+        public double Weight { get; set; }
+        public string Taste { get; set; }
+        public ushort CaloriesPerGram { get; set; }
+
+        public void ParameterOfPlan()
+        {
+            Name = Console.ReadLine();
+            Random rnd = new Random();
+
+            if (Name == "cucumber")
```

Comment:
I think it's better to create class of a concrete vegetable what will inherit BaseVegetable